src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs

```diff
@@ -193,22 +192,18 @@ private void LogException(LogLevel logLevel, IEnumerable<KeyValuePair<string, ob
                 LogTrace(logLevel, values, formattedMessage);
             }
 
-            ApplyScopeAndStateProperties(telemetry, values);
+            ApplyScopeAndStateProperties(telemetry.Properties, values);
 
             _telemetryClient.TrackException(telemetry);
         }
 
         private void LogTrace(LogLevel logLevel, IEnumerable<KeyValuePair<string, object>> values, string formattedMessage)
         {
-            TraceTelemetry telemetry = new TraceTelemetry(formattedMessage)
```

Comment:
Many of the signature changes above were to allow me to remove this object creation, and instead use the overload below that takes the values directly.